polyfills/URL/config.toml

```
@@ -32,3 +49,8 @@ op_mini = "*"
 safari = "<12.0"
 ios_saf = "<12.0"
 samsung_mob = "<5.0"
+
+[install]
+module = "whatwg-url"
+postinstall = "update.task.js"
+clean = [ "polyfill.js" ]
```

```suggestion
clean = [ "polyfill.js", "_webidl-conversions.js" ]
```

As both files are generated, both should be removed before running `update.task.js`?